### PR TITLE
Add #19200 improvement to the Order page specifications

### PR DESCRIPTION
**Pull request by LouiseBonnard**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Adds the tracking number in the 'Merchandise returns' tab of an order
| Fixed ticket? | Related to https://github.com/PrestaShop/PrestaShop/issues/19200#issuecomment-756072824

### CODE REVIEW

File: back-office/orders/orders/Order Page View.md [hunk 1]
```diff
@@ -516,7 +516,7 @@ The edit shipping button on each row can edit the information in a popin. You ca
 
 4.  **Merchandising returns tab**
 
-The merchandise return tab lists **the date, the tracking number, the type, Carrier (which is the return status) and the quantity**. When the merchandise return is disabled the tab displayed: "no merchandise returned yet".
+The merchandise return tab lists **the date, the tracking number, the type, the carrier (= return status), and the number (= return reference number)**. When the merchandise return is disabled the tab displayed: "no merchandise returned yet".
```
Comment (Robin-Fischer-PS) [inline back-office/orders/orders/Order Page View.md:519]: If I understand the issue well, the "tracking number" column is no longer displayed, and the "Quantity" column is replaced by the "number" column.

```suggestion
The merchandise return tab lists **the date, the type, the carrier (= return status), and the number (= return reference number)**. When the merchandise return is disabled the tab displayed: "no merchandise returned yet".
```

**Review by Robin-Fischer-PS — CHANGES_REQUESTED** [review]

File: back-office/orders/orders/Order Page View.md (hunk 1 above)
Comment (LouiseBonnard) [reply back-office/orders/orders/Order Page View.md:519]: Actually, there is no quantity column, I don't know why it was written in the specifications. @MatShir, any idea? As for the tracking number, it turned out to be the return ID, cf. https://github.com/PrestaShop/PrestaShop/issues/19200#issuecomment-638101058.

Comment (LouiseBonnard) [reply back-office/orders/orders/Order Page View.md]: @MatShir, fixed in this PR, I let you correct it for the order page specifications.

Comment (MatShir) [reply back-office/orders/orders/Order Page View.md]: I can confirm, it is the ID. It is a mistake from me when I started the specs.

**Review by MatShir — APPROVED** [review]

**Review by Robin-Fischer-PS — APPROVED** [review]

**Comment by marionf** [comment]
@LouiseBonnard Could you please resolve the conflicts ?

**Comment by marionf** [comment]
Thank you @LouiseBonnard :)

**Pull request merged**